Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,336 @@
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                    
+                    Text(model.nearbyFeaturesText)
+                        .foregroundColor(.orange)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .padding(.top, -24)
+                }
+                .padding(8)
+                .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .toolbar {
+                // Bottom button toolbar.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Current Section") {
+                        let sectionFeature = model.nearbyFeatures[model.currentSectionName!]!
+                        popup = Popup(geoElement: sectionFeature)
+                        isShowingPopup = true
+                    }
+                    .disabled(!model.hasCurrentSection)
+                    .opacity(isShowingPopup ? 0 : 1)
+                    
+                    Button("Point of Interest") {
+                        let poiFeature = model.nearbyFeatures[model.nearbyPOINames.first!]!
+                        popup = Popup(geoElement: poiFeature)
+                        isShowingPopup = true
+                    }
+                    .disabled(!model.hasPointOfInterest)
+                    .opacity(isShowingPopup ? 0 : 1)
+                }
+            }
+            .floatingPanel(
+                selectedDetent: .constant(.full),
+                horizontalAlignment: .leading,
+                isPresented: $isShowingPopup
+            ) {
+                Group {
+                    // Feature info popup.
+                    if let popup = popup {
+                        PopupView(popup: popup, isPresented: $isShowingPopup)
+                            .showCloseButton(true)
+                    }
+                }
+                .padding()
+            }
+            .task(id: isShowingPopup) {
+                // Stop location updates when the popup is showing.
+                if isShowingPopup {
+                    await model.locationDisplay.dataSource.stop()
+                } else {
+                    do {
+                        try await model.locationDisplay.dataSource.start()
+                    } catch {
+                        model.error = error
+                    }
+                }
+            }
+            .alert(isPresented: $model.isShowingAlert, presentingError: model.error)
+    }
+}
+
+private extension SetUpLocationDrivenGeotriggersView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A map of the Santa Barbara Botanic Garden.
+        let map = Map(item: PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: .santaBarbaraBotanicGardenMap
+        ))
+        
+        /// The location display for the map view.
+        lazy var locationDisplay = makeLocationDisplay()
```

Comment:
```suggestion
        let locationDisplay: LocationDisplay
```
Then change it here.